src/ioctls/vcpu.rs

```diff
@@ -1157,26 +1157,26 @@ impl VcpuFd {
     /// let vm = kvm.create_vm().unwrap();
     /// let vcpu = vm.create_vcpu(0).unwrap();
     ///
-    /// #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
-    /// {
-    ///     let debug_struct = kvm_guest_debug {
-    ///         // Configure the vcpu so that a KVM_DEBUG_EXIT would be generated
-    ///         // when encountering a software breakpoint during execution
-    ///         control: KVM_GUESTDBG_ENABLE | KVM_GUESTDBG_USE_SW_BP,
-    ///         pad: 0,
-    ///         // Reset all x86-specific debug registers
-    ///         arch: kvm_guest_debug_arch {
-    ///             debugreg: [0, 0, 0, 0, 0, 0, 0, 0],
-    ///         },
-    ///     };
+    /// let debug_struct = kvm_guest_debug {
```

Comment:
You also enable this for all other architectures such as s390 and ppc, which may not be what you intended.

Reply:
Good point, thanks for the review. I updated this PR and I think the latest implementation excluded the s390 and ppc.